### PR TITLE
Add wbw highlighting

### PR DESCRIPTION
**Pull request by aabdurrahman**
This adds ability to select and highlight individual words

### CODE REVIEW

File: app/src/main/java/com/quran/labs/androidquran/presenter/quran/ayahtracker/AyahTrackerPresenter.kt [hunk 1]
```diff
@@ -17,9 +20,7 @@ import com.quran.labs.androidquran.presenter.Presenter
 import com.quran.labs.androidquran.presenter.quran.ayahtracker.AyahTrackerPresenter.AyahInteractionHandler
 import com.quran.labs.androidquran.ui.PagerActivity
 import com.quran.labs.androidquran.ui.helpers.AyahSelectedListener.EventType
-import com.quran.labs.androidquran.ui.helpers.AyahSelectedListener.EventType.DOUBLE_TAP
-import com.quran.labs.androidquran.ui.helpers.AyahSelectedListener.EventType.LONG_PRESS
-import com.quran.labs.androidquran.ui.helpers.AyahSelectedListener.EventType.SINGLE_TAP
+import com.quran.labs.androidquran.ui.helpers.AyahSelectedListener.EventType.*
```
Comment (ahmedre) [inline app/src/main/java/com/quran/labs/androidquran/presenter/quran/ayahtracker/AyahTrackerPresenter.kt:23]: no star imports please :)

Comment (aabdurrahman) [reply app/src/main/java/com/quran/labs/androidquran/presenter/quran/ayahtracker/AyahTrackerPresenter.kt]: fixed

File: app/src/main/java/com/quran/labs/androidquran/presenter/quran/ayahtracker/AyahTrackerPresenter.kt [hunk 2]
```diff
@@ -288,6 +289,20 @@ class AyahTrackerPresenter @Inject constructor(
...
+  }
+
+  private fun getGlyphForPosition(page: Int, x: Float, y: Float): AyahGlyph? {
+    for (item in items) {
+      val glyph = item.getGlyphForPosition(page, x, y)
+      if (glyph != null) {
+        return glyph
+      }
+    }
+    return null
```
[the remark this answers is not in the document]
Comment (aabdurrahman) [reply app/src/main/java/com/quran/labs/androidquran/presenter/quran/ayahtracker/AyahTrackerPresenter.kt]: fixed

**Comment by ahmedre** [comment]
jazakumAllah khairan!

**Pull request merged**